app/routes/_layout+/bookings.$bookingId.tsx

```diff
@@ -172,14 +162,25 @@ export async function action({ request, params }: ActionFunctionArgs) {
           }
         );
       }
-      const { name, startDate, endDate, custodian } = result.data;
+
+      const { name, custodian } = result.data;
+      const hints = getHints(request);
+      const startDate = formData.get("startDate")!.toString();
+      const endDate = formData.get("endDate")!.toString();
+      const fmt = "yyyy-MM-dd'T'HH:mm";
+      const from = DateTime.fromFormat(startDate, fmt, {
+        zone: hints.timeZone,
+      }).toJSDate();
+      const to = DateTime.fromFormat(endDate, fmt, {
+        zone: hints.timeZone,
+      }).toJSDate();
```

Comment:
this is necessary as `result.data.startDate` will have booking time overriden(not converted) to server timezone.

Reply:
Clear!